model/lifecycle.go

```diff
@@ -1771,6 +1771,34 @@ func addNewTasks(ctx context.Context, creationInfo TaskCreationInfo, existingBui
 	return activatedTaskIds, nil
 }
 
+// activateExistingInactiveTasks will find existing inactive tasks in the patch that need to be activated as
+// part of the patch re-configuration.
+func activateExistingInactiveTasks(creationInfo TaskCreationInfo, existingBuilds []build.Build) error {
+	for _, b := range existingBuilds {
+		tasksInBuild, err := task.FindAll(db.Query(task.ByBuildId(b.Id)))
```

Comment:
can we readd withFields displayName and Activated? Really it'd be nice if we could refactor out the call to find existingTasks and pass it into addNewTasks and activateExistingInactiveTasks but it may not be worth it for this relatively inexpensive query.

Reply:
Sure, `BuildIdKey` and `VersionKey` need to be added as well for `SetActiveState` to work, I'll add those too.